config/routes.rb

```diff
@@ -3,6 +3,11 @@
 
   resources :items do
     patch :update_lending
+    collection do
+      get "/my", to: "items#my_items"
+      get "/my/borrowed", to: "items#my_borrowed_items"
+      get "/borrowed", to: "items#borrowed_items"
```

Comment:
- structure: please add aliases (`as: :some_label`) for these routes, so that they can be addressed validly within the templates